app/views/facility_estimates/_form.html.haml

```diff
@@ -0,0 +1,55 @@
+= simple_form_for @estimate, url: form_url, method: form_method do |f|
+  .inline-form-controls
+    = f.input :name
+    .span-4
+      = f.label :user_id, Estimate.human_attribute_name(:user_id)
+      = f.input_field :user_id,
+        collection: user_collection,
+        class: "js--chosen js--select-with-search",
+        include_blank: false,
+        data: { search_url: search_facility_estimates_path },
+        disabled: f.object.persisted?
```

Comment:
Why is the user field disabled? According to the ticket all fields should be editable.